otsserver/stamper.py

```diff
@@ -420,7 +420,18 @@ def __do_bitcoin(self):
                 logging.error("Maximum txfee reached!")
                 return
 
-            r = proxy.signrawtransaction(unsigned_tx)
+            # before to sign check the btc version
+            # signrawtransaction is deprecated from 0.17.x
+            # and removed from 0.18.x
+            try:
+                r = proxy.call('getnetworkinfo')
+                self.btc_version = int(r['version'])
+            except bitcoin.rpc.JSONRPCError as err:
+                logging.debug("Error calling getnetworkinfo: %r" % err.error)
+            if self.btc_version >= 170000:
+                r = proxy.signrawtransactionwithwallet(unsigned_tx)
+            else:
+                r = proxy.signrawtransaction(unsigned_tx)
```

Comment:
I'd recommend instead just updating the minimum required version in the README. It's OK if `otsd` has fairly specific compatibility requirements.

Reply:
I had not seen the change Riccardo made in the README when I wrote these lines... New commit simplified.